app/src/main/java/com/github/polypoly/app/ui/game/GameActivity.kt

```diff
@@ -44,36 +42,39 @@ class GameActivity : ComponentActivity() {
     @Composable
     fun GameActivityContent() {
         val player = gameModel.getPlayerData().observeAsState().value
+        mapViewModel.currentPlayer = player!!
```

Comment:
Why you need to use `!!` here? `gameModel.getPlayerData().observeAsState().value` can be null? In which situation it is null?